lib/proxy/log_buffer/ring_buffer.rb

```diff
@@ -0,0 +1,56 @@
+module Proxy::LogBuffer
```

Comment:
No copyright notice that this uses code from the celluloid project; MIT license under which celluloid is distributed requires attribution.